src/test/kotlin/org/rust/ide/intentions/InjectLanguageIntentionTest.kt

```diff
@@ -0,0 +1,73 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.intentions
+
+import com.intellij.injected.editor.EditorWindow
+import com.intellij.lang.injection.InjectedLanguageManager
+import com.intellij.openapi.editor.Editor
+import com.intellij.psi.PsiFile
+import com.intellij.psi.PsiLanguageInjectionHost
+import com.intellij.psi.injection.Injectable
+import org.intellij.lang.annotations.Language
+import org.intellij.plugins.intelliLang.inject.InjectLanguageAction
+import org.intellij.plugins.intelliLang.inject.InjectedLanguage
+import org.rust.lang.core.psi.ext.ancestorOrSelf
+
+class InjectLanguageIntentionTest : RsIntentionTestBase(InjectLanguageAction()) {
+    // It's external intention, we don't have documentation
+    override fun `test intention has documentation`() {}
```

Comment:
No need anymore because intention documentation test moved into `https://github.com/intellij-rust/intellij-rust/blob/master/src/test/kotlin/org/rust/ide/intentions/RsIntentionDocumentationTest.kt`